operations/mimir-mixin/docs/playbooks.md

```diff
@@ -1102,6 +1102,98 @@ An exemplar must have a valid timestamp, otherwise it cannot be correlated to an
 
 > **Note**: Invalid exemplars are skipped during the ingestion, and valid exemplars within the same request are ingested.
 
+### err-mimir-ingester-max-ingestion-rate
+
+This critical error occurs when the rate of received samples per second is exceeded in an ingester.
+
+The ingester implements a rate limit on the samples per second that can be ingested, and it's used to protect an ingester from overloading in case of an high traffic.
+The limit is a per-instance limit and it's applied on all samples received, across all tenants, in each ingester.
+
+How to **fix**:
+- Scale up ingesters.
+- Increase the limit by using the `-ingester.instance-limits.max-ingestion-rate` option (or `max_ingestion_rate` in the runtime config).
+
+### err-mimir-ingester-max-tenants
+
+This critical error occurs when the ingester receives a write request for a new tenant (a tenant for which no series have been stored yet) but the ingester cannot accept it because the maximum number of allowed tenants per ingester has been reached.
+
+How to **fix**:
+- In case of emergency, increase the limit by using the `-ingester.instance-limits.max-tenants` option (or `max_tenants` in the runtime config).
+- Consider configuring ingesters shuffle sharding to reduce the number of tenants per ingester.
+
+### err-mimir-ingester-ingester-max-series
+
+This critical error occurs when an ingester rejects a write request because it reached the maximum number of in-memory series.
+
+How it **works**:
+
+- The ingester keeps most recent series data in-memory.
+- The ingester has a per-instance limit on the number of in-memory series, used to protect the ingester from overloading in case of an high traffic.
+- When the number of in-memory series, new series are rejected, while samples can still be appended to existing ones.
+- You can configure the limit by setting the `-ingester.instance-limits.max-series` option (or `max_series` in the runtime config).
+
+How to **fix**:
+- See [`MimirIngesterReachingSeriesLimit`](#MimirIngesterReachingSeriesLimit) runbook.
+
+### err-mimir-ingester-max-inflight-push-requests
+
+This error occurs when an ingester rejects a write request because the max inflight requests limit has been reached.
+
+How it **works**:
+- The ingester has per-instance limit on the number of inflight write (push) requests.
+- The limit applies on all inflight write requests, across all tenants, and is used to protect the ingester from overloading in case of an high traffic.
```

Comment:
```suggestion
- The limit applies on all inflight write requests, across all tenants, and is used to protect the ingester from overloading in case of high traffic.
```